test/tests/steel_creep_damage_oh/power_law_creep.i

```diff
@@ -0,0 +1,189 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  second_order = true
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  volumetric_locking_correction = false
+[]
+
+[AuxVariables]
+  [local_damage]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [omega]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [combined_creep_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[AuxKernels]
+  [local_damage]
+    type = MaterialRealAux
+    variable = local_damage
+    property = local_damage
+    execute_on = TIMESTEP_END
+  []
+  [omega]
+    type = MaterialRealAux
+    variable = omega
+    property = omega
+  []
+  [combined_creep_strain_yy]
+    type = RankTwoAux
+    variable = combined_creep_strain_yy
+    rank_two_tensor = combined_creep_strain
+    index_j = 1
+    index_i = 1
+    execute_on = timestep_end
+  []
+[]
+
+[Functions]
+  [./pull]
+    type = PiecewiseLinear
+    x = '0 10'
+    y = '0 1e-3'
+  [../]
+[]
+
+[Modules/TensorMechanics/Master]
+  [./all]
+    strain = FINITE
+    add_variables = true
+    use_finite_deform_jacobian = true
+    generate_output = 'hydrostatic_stress'
+  [../]
+[]
+
+[Materials]
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 1e10
+    poissons_ratio = 0.3
+  [../]
+
+  [./elastic_strain]
+    type = ComputeMultipleInelasticStress
+    # tangent_operator = nonlinear
+    damage_model = 'steel_local'
+    inelastic_models='creep_ten creep_ten2'
+  [../]
+
+  [steel_local]
+    type = SteelCreepDamageOh
+    epsilon_f = 0.07
+    creep_law_exponent = 10
+    use_displaced_mesh = true
+    damage_index_name = local_damage
+    reduction_damage_threshold = 0.5
+    creep_strain_names='creep_ten_creep_strain creep_ten2_creep_strain'
+    use_old_damage = true
+  []
+
+  [./creep_ten]
+    type = PowerLawCreepStressUpdate
+    coefficient = 10e-24
+    n_exponent = 4
+    activation_energy = 0
+    base_name = creep_ten
+  [../]
+  [./creep_ten2]
+    type = PowerLawCreepStressUpdate
+    coefficient = 10e-24
+    n_exponent = 4
+    activation_energy = 0
+    base_name = creep_ten2
+  [../]
+  [./creep_one]
```

Comment:
It might be helpful to add a comment to this one:
#This model is activated with cli_args to compare the response with a single model with that of two models

Reply:
Added the comment. Also deleted the existing populated field in the original input file. The file now is activated by `cli_args`.